arviz/data/io_dict.py

```diff
@@ -156,6 +169,26 @@ def constant_data_to_xarray(self):
             constant_data[key] = xr.DataArray(vals, dims=val_dims, coords=coords)
         return xr.Dataset(data_vars=constant_data, attrs=make_attrs(library=None))
 
+    @requires("predictions_constant_data")
+    def predictions_constant_data_to_xarray(self):
```

Comment:
If it is not too much to ask, no pressure, could you do the same trick here as with constant data and predictions constant data in from_pyro and from_numpyro? If I am not mistaken, the code here is actually shared not between the 2 constant data group but also with observed data group, it would significantly reduce code duplication

Reply:
Of course! It makes much more sense. Thanks!

Reply:
It could be done, I am not sure it is worth it though, they are already 3 lines long, the bulk of the code is already the external function `dict_to_dataset`, the only difference between them is the error.